src/webargs/tornadoparser.py

```diff
@@ -57,8 +57,8 @@ def __getitem__(self, key):
                 return _unicode(value)
             return value
         # based on tornado.web.RequestHandler.decode_argument
-        except UnicodeDecodeError:
-            raise HTTPError(400, f"Invalid unicode in {key}: {value[:40]!r}")
+        except UnicodeDecodeError as exc:
+            raise HTTPError(400, f"Invalid unicode in {key}: {value[:40]!r}") from exc
```

Comment:
I'm just curious, was pyupgrade the tool that flagged `raise` vs `raise ... from`? I see a few of these, so I assume it was part of the linting update.

Reply:
IIRC, flake8 complained.

Reply:
Ah, I didn't notice the change in `flake8-bugbear`. I see now they introduced a new error, `B904` for this. Cool!